vhdbuilder/packer/install-dependencies.sh

```diff
@@ -468,6 +468,7 @@ MULTI_ARCH_KUBE_BINARY_VERSIONS="
 1.23.3-hotfix.20220401
 1.23.4-hotfix.20220331
 1.23.5-hotfix.20220331
+1.24.0-beta.0
```

Comment:
Seems like this needs a beta**.0**, which should not be needed for other k8s binaries. Tested with URL.

Reply:
cc: @AbelHu